src/entities/DealTokenSwap.ts

```diff
@@ -204,6 +221,42 @@ export class DealTokenSwap implements IDeal {
     return this.fundingWasInitiated && !this.isExecuted && !this.fundingPeriodHasExpired;
   }
 
+  /**
+   * Gets the DAO based on the connected wallet address
+   */
+  @computedFrom("ethereumService.defaultAccountAddress")
+  public get daoRelatedToWallet(): IDAO{
+    return this.getDao(true);
+  }
+
+  /**
+    * Gets the non-related DAO based on the connected wallet address
+    */
+  @computedFrom("ethereumService.defaultAccountAddress")
+  public get otherDao(): IDAO {
+    return this.getDao(false);
+  }
+
+  /**
+    * Gets the DAO based on the connected wallet address
```

Comment:
Would be nice to have docs on relatedToWallet param, or rename it to be more clear

Reply:
Additional comments added